test/get_previous_releases.py

```diff
@@ -5,7 +5,7 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 #
 # Download or build previous releases.
-# Needs curl and tar to download a release, or the build dependencies when
+# Needs urllib built-in python library and tar to download a release, or the build dependencies when
```

Comment:
If this is part of the standard library, I don't think you need to list it as a requirement.